index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <!-- CSS Stylesheet-->
+        <link rel="stylesheet" type="text/css" href="src/css/style.css">
+        <!--Google fonts-->
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;600;700&display=swap" rel="stylesheet">
+        <!-- Font Awesome icons -->
+        <script src="https://kit.fontawesome.com/0e63fe95a0.js" crossorigin="anonymous"></script>
+        <title>Anghelo's Portfolio</title>
+    </head>
+    <body>
+        <nav class="nav-container">
+            <h4 class="nav-wordsColor">Luis Abarca</h4>
```

Comment:
- [ ] Ensure that your logo is a link. Here it is an `h4` tag but it should be wrapped in a `a` tag instead.